fast_response/plotting_utils.py

```diff
@@ -100,14 +103,44 @@ def plot_labels(src_dec, src_ra, reso):
     plt.text(np.radians(-reso),np.radians(-1.75*reso), r"%.1f$^{\circ}$"%(reso+np.degrees(src_ra)),
              horizontalalignment='center',
              verticalalignment='top', fontsize=fontsize)
-    plt.text(-1*np.radians(2.35*reso), np.radians(0), r"declination", 
+    plt.text(-1*np.radians(2.35*reso*label_scale), np.radians(0), r"declination", 
                 ha='center', va='center', rotation=90, fontsize=fontsize)
-    plt.text(np.radians(0), np.radians(-2.05*reso), r"right ascension", 
+    plt.text(np.radians(0), np.radians(-2.05*reso*label_scale), r"right ascension", 
                 ha='center', va='center', fontsize=fontsize)
+def compute_ang_err(ra,dec,sigma):
+        dec = np.pi/2 - dec
+        sigma = np.rad2deg(sigma)
+        delta, step, bins = 0, 0, 0
+        delta= sigma/180.0*np.pi
+        step = 1./np.sin(delta)/20.
+        bins = int(360./step)
+        Theta = np.zeros(bins+1, dtype=np.double)
+        Phi = np.zeros(bins+1, dtype=np.double)
+        # define the contour
+        for j in range(0,bins):
+                phi = j*step/180.*np.pi
+                vx = np.cos(phi)*np.sin(ra)*np.sin(delta) + np.cos(ra)*(np.cos(delta)*np.sin(dec) + np.cos(dec)*np.sin(delta)*np.sin(phi))
+                vy = np.cos(delta)*np.sin(dec)*np.sin(ra) + np.sin(delta)*(-np.cos(ra)*np.cos(phi) + np.cos(dec)*np.sin(ra)*np.sin(phi))
+                vz = np.cos(dec)*np.cos(delta) - np.sin(dec)*np.sin(delta)*np.sin(phi)
+                Theta[j], Phi[j] = hp.vec2ang(np.array([vx, vy, vz]))
+
+        Theta[bins] = Theta[0]
+        Phi[bins] = Phi[0]
+
+        return Theta, Phi
+def plot_events2(dec,ra,sigmas, src_ra, src_dec, reso, sigma_scale=5., col = 'k', constant_sigma=False,
```

Comment:
could you choose a more descriptive name here? just so it's clear what the difference between this and just `plot_events` is